compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -499,6 +499,7 @@ def IREELinalgExt_AttentionOp : IREELinalgExt_PureOp<"attention",
                        AnyShaped:$output,
                        AffineMapArrayAttr:$indexing_maps
   );
+  let regions = (region SizedRegion<1>:$region);
```

Comment:
Can we give these operations this trait https://mlir.llvm.org/docs/Traits/#single-block-region ?